package/Dockerfile

```diff
@@ -22,8 +22,9 @@ RUN if [ "$TARGETPLATFORM" != "linux/amd64" ] && [ "$TARGETPLATFORM" != "linux/a
 
 ENV ARCH ${TARGETPLATFORM#linux/}
 
-RUN zypper -n addrepo --refresh https://download.opensuse.org/repositories/system:/snappy/SLE_15/system:snappy.repo && \
-    zypper -n addrepo --refresh https://download.opensuse.org/repositories/network:/utilities/15.6/network:utilities.repo && \
+RUN zypper -n addrepo --refresh https://download.opensuse.org/repositories/system:/snappy/openSUSE_Factory/system:snappy.repo && \
```

Comment:
Do we also need to update the `Dockerfile.dapper` file?

https://github.com/longhorn/backing-image-manager/blob/7a1731de8933705a970572ed94c50205a10651f5/Dockerfile.dapper#L17